code/modules/antagonists/heretic/knowledge/ash_lore.dm

```diff
@@ -26,19 +26,36 @@
  *
  * Ashlord's Rite
  */
+
+/datum/heretic_knowledge_tree_column/main/ash
+	id = HKT_UUID_ASH
+	neighbour_id_0 = HKT_UUID_COSMIC_TO_ASH
+	neighbour_id_1 = HKT_UUID_ASH_TO_MOON
```

Comment:
Hm, do we actually *need* to define uuids for these things?

Can't we just use their typepath? It's unique to them, after all

Like
```dm
/datum/heretic_knowledge_tree_column/main/ash
	// id = /datum/heretic_knowledge_tree_column/main/ash // can just access this with .type
	neighbour_id_0 = /datum/heretic_knowledge_tree_column/cosmic_to_ash
	neighbour_id_1 = /datum/heretic_knowledge_tree_column/ash_to_moon

Reply:
It'd be a lot more ergonomic if you didn't have to define any ids, and could just slap your typepath down and it'd work out of the box

Reply:
yeah seems sensible